src/mscorlib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -487,7 +489,10 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Span.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Span.Fast.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\SpanDebugView.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)System\Span.NonGeneric.cs" />
```

Comment:
I think you will need to keep the non-generic Span class temporary (with forwarders to the new implementations), so that things are not on the floor.

Reply:
Even after this change https://github.com/dotnet/coreclr/pull/16520 propagates completely (which has these temporary APIs https://github.com/dotnet/coreclr/pull/16520/files#diff-34f82797d649dc710c6ff9268fa79c9bR691)?

Reply:
I think so.

Reply:
@jkotas, I have added back Span.NonGeneric but only kept a small subset of the APIs (specifically the ones that were recently renamed). Is that sufficient or do I need to add back all old APIs with forwarders to the impl in MemoryExtensions?

Reply:
You just need to add enough to make CoreCLR CI green. If the CoreCLR CI is green with what you go, then it is fine.

Reply:
It looks like there are several tests that end up calling Span.AsBytes and are failing.

I can't keep both Span.AsBytes and MemoryExtensions.AsBytes since they are extension methods, and would conflict, so should I rename AsBytes in MemoryExtensions for now? Or should I try to temporarily disable the subset of tests that end up calling AsBytes?

1) Rename AsBytes, to something like AsBytesExtension, in MemoryExtensions
2) Update use of AsBytes in corefx to AsBytesExtension
3) Remove Span.AsBytes and add MemoryExtensions.AsBytes
4) Update use of AsBytesExtension in corefx to AsBytes
5) Remove AsBytesExtension.

Is there a simpler approach?

Uses of AsBytes in corefx:
https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/src/System/Xml/NameTable.cs - ComputeHash32
https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/src/System/Net/IPAddress.cs - GetHashCode

Reply:
You should keep `Span.AsBytes` forwarder but not make non-extension method:

```
public static class Span
{
    public static Span<byte> AsBytes<T>(Span<T> source) => source.AsBytes();
}
```